src/conv.jl

```diff
@@ -161,3 +163,19 @@ if is_nnpack_available()
         return conv_nnpack(x, w, cdims; kwargs...)
     end
 end
+
+function conv(x, w::AbstractArray{T, N}; stride = 1, pad = 0, dilation = 1) where {T, N}
+    stride = expand(Val(N-2), stride)
+    pad = expand(Val(N-2), pad)
+    dilation = expand(Val(N-2), dilation)
+    cdims = DenseConvDims(x, w; stride = stride, padding = pad, dilation = dilation)
+    return conv(x, w, cdims)
+end
+
+function depthwiseconv(x, w::AbstractArray{T, N}; stride = 1, pad = 0, dilation = 1) where {T, N}
```

Comment:
I think you're missing the `flipped` boolean parameter.

Reply:
Done. I have added the flipped parameter. Thank you!